subsys/logging/Kconfig

```diff
@@ -302,6 +302,36 @@ config LOG_BACKEND_UART
 	help
 	  When enabled backend is using UART to output logs.
 
+config HAS_SWO
```

Comment:
this is SoC feature. I'm not 100% sure where to put it, but maybe here:
https://github.com/zephyrproject-rtos/zephyr/blob/master/soc/arm/Kconfig

Like MPU is also an option for ARM core and it's there.

Reply:
Thanks, I've moved the hidden option as suggested.